services/web/server/src/simcore_service_webserver/exporter/_formatter/template_json.py

```diff
@@ -0,0 +1,22 @@
+import json
+from pathlib import Path
+
+import aiofiles
+
+from ...projects.models import ProjectDict
+
+FILE_NAME = "template.json"
+
+
+def get_content(project_data: ProjectDict) -> str:
+    data = {
+        "uuid": project_data["uuid"],
+        "name": project_data["name"],
+        "description": project_data["description"],
+    }
+    return json.dumps(data)
```

Comment:
MINOR: recommend to use `servicelib.json_serialization.json_dumps` instead

https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/service-library/src/servicelib/json_serialization.py